File: NewsFlash_edited.Models - Copy/models/Article.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace NewsFlash_edited.Models.models
{
  public class Article
  {
    [Key]
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public int Id { get; set; }
    public string Header { get; set; }
    public string Summary { get; set; }
    public string Img { get; set; }
    public string Body { get; set; }
    public string Link { get; set; }
    public string Source { get; set; }
    public DateTime? Publishdate { get; set; }
    public virtual ICollection<ArticleTag> ArticleTag { get; set; }

    //public bool IsPublished { get { return this.Publishdate.HasValue; } }
  }

}

File: NewsFlash_edited - Copy/models/article.ts
  export class Article {
  public id: number;
  public header: string;
  public img: string;
  public summary: string;
  public body: string;
  public link: string;
  public source: string;
  public publishDate: Date;
  public tags: string[];

  constructor(id: number, header: string, img: string, summary: string, body: string, link: string, source: string, publishDate: Date, tags: string[])
  {
    this.id = id;
    this.header = header;
    this.img = img;
    this.summary = summary;
    this.body = body;
    this.link = link;
    this.source = source;
    this.publishDate = publishDate;
    this.tags = tags;
  }

}
